homeassistant/components/sensor/const.py

```diff
@@ -194,6 +205,12 @@ class SensorDeviceClass(StrEnum):
     Unit of measurement: `Wh`, `kWh`, `MWh`, `MJ`, `GJ`
     """
 
+    FORCE = "force"
```

Comment:
A new device class requires approval in an architecture discussion. Please limit the discussion to one device class at a time.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Thanks. I'm new to contributing. So I apologize.